include/bluetooth/mesh/gen_onoff_srv.h

```diff
@@ -98,6 +99,8 @@ struct bt_mesh_onoff_srv {
 	struct bt_mesh_model *model;
 	/** Publish parameters. */
 	struct bt_mesh_model_pub pub;
+	/* Scene entry */
+	struct bt_mesh_scene_entry scene;
```

Comment:
They way this is set up, then this struct take zero space, right?

Reply:
Yes